### PR TITLE
Fix issue #10

### PR DESCRIPTION
**Pull request by serluca**
Fix the crash when using cocoapods framework, see #10

### CODE REVIEW

File: Source/PTEDashboard.m [hunk 1]
```diff
@@ -59,9 +59,9 @@ - (instancetype)initWithFrame:(CGRect)frame
         }
         
         // Load Storyboard
-        self.rootViewController = [[UIStoryboard storyboardWithName:@"LumberjackConsole"
-                                                             bundle:nil] instantiateInitialViewController];
-        
+			  UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"LumberjackConsole" bundle:[NSBundle bundleForClass:[self class]]];
+			  self.rootViewController = [storyboard instantiateInitialViewController];
+
```
Comment (rivera-ernesto) [inline Source/PTEDashboard.m]: Looks good. Just what happened to the indentation?

File: Source/PTEDashboard.m [hunk 2]
```diff
@@ -106,14 +106,14 @@ - (void)handleStatusBarOrientationChange:(NSNotification *)notification
...
-    }
-    
+	  CGRect frame = self.frame;
+	  if ((UIInterfaceOrientationIsLandscape(currentOrientation) && UIInterfaceOrientationIsPortrait(nextOrientation)) ||
+			(UIInterfaceOrientationIsPortrait(currentOrientation) && UIInterfaceOrientationIsLandscape(nextOrientation)))
+	  {
+		  frame.size = CGSizeMake(frame.size.height,
+														frame.size.width);
+	  }
+
```
Comment (rivera-ernesto) [inline Source/PTEDashboard.m]: Same here.

File: Source/PTEDashboard.m (hunk 1 above)
Comment (serluca) [inline Source/PTEDashboard.m]: Sorry, probably we are using different indentation settings

Comment (rivera-ernesto) [inline Source/PTEDashboard.m]: Yes, probably mines are the default ones. Could you just fix them so I can merge it?

Comment (serluca) [inline Source/PTEDashboard.m]: Should be ok now, the problem is that it is perfectly indented in my Xcode. 
Xcode automatically align the indentation

**Comment by rivera-ernesto** [comment]
Seems that your problem is that your indentation uses tabs instead of spaces, so depending on the editor it will always look off (the reason why tabs are discouraged.

**Comment by serluca** [comment]
Is the last commit still not good?
Can you give me your indentation settings? I don't know how to do, it looks perfectly aligned for me

File: Source/PTEDashboard.m [hunk 3]
```diff
@@ -106,14 +106,13 @@ - (void)handleStatusBarOrientationChange:(NSNotification *)notification
...
-        frame.size = CGSizeMake(frame.size.height,
-                                frame.size.width);
-    }
-    
+	CGRect frame = self.frame;
+	if ((UIInterfaceOrientationIsLandscape(currentOrientation) && UIInterfaceOrientationIsPortrait(nextOrientation)) ||
+		(UIInterfaceOrientationIsPortrait(currentOrientation) && UIInterfaceOrientationIsLandscape(nextOrientation)))
+	{
+		frame.size = CGSizeMake(frame.size.height,frame.size.width);
+	}
```
Comment (rivera-ernesto) [inline Source/PTEDashboard.m]: Drag your mouse in your indentation code here and you'll notice that it indents with either tab for each level.

Regular Xcode indentation (so this and most iOS/OSX projects) uses four spaces for each level of indentation.

You can take a look here at how to change the setting and some discussion about indentation ;)

http://stackoverflow.com/a/3336065/1049134

Comment (rivera-ernesto) [inline Source/PTEDashboard.m]: Also you can see [here](https://github.com/PTEz/LumberjackConsole/pull/11/files) how your edits don't look indented in GitHub (and so in many editors that don't represent a tab with four spaces).

**Comment by serluca** [comment]
What do you think about the last commit? Is it aligned?

**Comment by rivera-ernesto** [comment]
Yes, looks really good now ;)

Thanks for you time.

**Pull request merged**